ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -224,7 +228,7 @@
         subscriptions
          (->> (mdb/find-concepts context {:latest true} :subscription)
               (filter #(not (:deleted %)))
-              (map #(select-keys % [:email-address :metadata :collection-concept-id])))]
+              (map #(select-keys % [:extra-fields :metadata])))]
     ;; for some reason, the info or println doesn't show inside the for loop in my local
     ;; proto repl.
     (println (process-subscriptions context subscriptions time-constraint))))
```

Comment:
replace println with info. for is lazily evaluated which is why you don't see it being logged. Change your `for` to a `doseq` or wrap it in a `doall` and you will see it logged.

Reply:
info didn't work, so I used println. will change it when I add the real send email function in another ticket.